.github/workflows/test.yml

```diff
@@ -52,10 +61,26 @@ jobs:
         -DOPENASSETIO_ENABLE_SANITIZER_ADDRESS=ON 
         -DOPENASSETIO_ENABLE_SANITIZER_UNDEFINED_BEHAVIOR=ON
 
-    - name: Build, install and test
+    - name: Configure CMake build (Windows)
+      if: runner.os == 'Windows'
+      run: |
+        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+        cmake -G Ninja -S . -B build --install-prefix %GITHUB_WORKSPACE%\dist --toolchain %USERPROFILE%\conan\conan_paths.cmake -DCMAKE_VERBOSE_MAKEFILE=ON -DOPENASSETIO_ENABLE_TESTS=ON -DOPENASSETIO_ENABLE_SANITIZER_ADDRESS=ON -DOPENASSETIO_ENABLE_SANITIZER_UNDEFINED_BEHAVIOR=ON
```

Comment:
[fix] 
* Remove `*_SANITIZER_*` variables, they're unix-only.
* Would be good to line-wrap if possible.

Reply:
Fixed with https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/244/commits/a7c407c8f47bad76a4174cb319c3d1d502208e90 and https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/244/commits/c0d664c202ca31e383ddf68784f0b6c48abed789

Reply:
Interesting usage of `>`, I didn't realise that you can break up the line wrapping by adding a blank line, really nice.